src/sandstorm/run-bundle.c++

```diff
@@ -205,6 +205,42 @@ struct UserIds {
   kj::Array<gid_t> groups;
 };
 
+kj::Maybe<kj::Array<uint>> getPorts(uint httpsPort, kj::StringPtr portList) {
+    // TODO: Whose responsibility is it to bail out if httpsPort shows up
```

Comment:
An exception will presumably be thrown down the road due to the port being in-use, which I think is good enough?